enterprise/api/v1/views.py

```diff
@@ -455,6 +461,37 @@ class EnterpriseCustomerReportingConfigurationViewSet(EnterpriseReadOnlyModelVie
     filter_fields = FIELDS
     ordering_fields = FIELDS
 
+    @permission_required(
+        'enterprise.can_manage_reporting_config',
+        fn=lambda request, *args, **kwargs: get_ent_cust_from_report_config_uuid(kwargs['uuid']))
+    def retrieve(self, request, *args, **kwargs):
+        return super(EnterpriseCustomerReportingConfigurationViewSet, self).retrieve(request, *args, **kwargs)
+
+    @permission_required(
+        'enterprise.can_manage_reporting_config',
+        fn=lambda request, *args, **kwargs: get_enterprise_customer_from_user_id(request.user.id))
+    def create(self, request, *args, **kwargs):
+        config_data = request.data
+        enterprise_customer = models.EnterpriseCustomerUser.objects.get(user_id=request.user.id).enterprise_customer
+        config_data['enterprise_customer_id'] = str(enterprise_customer.uuid)
```

Comment:
Can we use the `get_enterprise_customer_from_user_id` util you created for these two lines?